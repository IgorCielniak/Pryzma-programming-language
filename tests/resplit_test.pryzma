// Split a string by commas
a = "apple,banana,cherry"
b = resplit(",", a)
print b  // Output: ["apple", "banana", "cherry"]

// Split a string by whitespace
c = "hello world"
d = resplit("\s+", c)
print d  // Output: ["hello", "world"]

// Split a string by a custom regex pattern
e = "123abc456def"
f = resplit("\d+", e)
print f  // Output: ["", "abc", "def"]
