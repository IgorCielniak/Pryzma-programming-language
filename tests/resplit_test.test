python Pryzma.py tests/resplit_test.pryzma
