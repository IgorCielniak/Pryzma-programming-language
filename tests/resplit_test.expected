['apple', 'banana', 'cherry']['hello', 'world']['', 'abc', 'def']
